src/app/containers/WebVitals/index.jsx

```diff
@@ -2,67 +2,13 @@
 import { useContext } from 'react';
 import useWebVitals from '@bbc/web-vitals';
 import useToggle from '#hooks/useToggle';
-import { FRONT_PAGE } from '#app/routes/utils/pageTypes';
-import {
-  CPS_LONG_ARTICLE_WITH_PNG_AND_VISJO,
-  CPS_LONG_ARTICLE_YOUTUBE_EMBED,
-  CPS_MEDIUM_ARTICLE,
-  CPS_SHORT_ARTICLE_YOUTUBE_EMBED,
-  CPS_SHORT_ARTICLE,
-  HOMEPAGE_LONG,
-  HOMEPAGE_SHORT,
-  MEDIA_ASSET_AUDIO,
-  MEDIA_ASSET_VIDEO,
-  RADIO_PAGE,
-  PODCAST_PAGE,
-} from './pageTypes';
 
 // Contexts
 import { UserContext } from '#contexts/UserContext';
-import { RequestContext } from '#contexts/RequestContext';
 
-const getPageType = ({ pathname, pageType }) => {
```

Comment:
Are you sure that we are removing the specific urls at this point? I understood that we were keeping these for CPS/Home/MAP performance tracking as these urls receive enough traffic

Reply:
Yep after a conversation with @gavinspence we decided to remove these specific urls from being tracked. I don't think some of them even show in the dashboard. Happy to change tact and keep them if you think so

Reply:
Yep - most of the URLs don't even show on the dashboard. Hoping that page reporting will be more representative

Reply:
If we think that this will be more representative I have no complains 😄